egs/wsj/s5/steps/online/nnet3/decode.sh

```diff
@@ -0,0 +1,154 @@
+#!/bin/bash
+
+# Copyright 2014  Johns Hopkins University (Author: Daniel Povey)
+#           2016  Api.ai (Author: Ilya Platonov)
+# Apache 2.0
+
+# Begin configuration section.  
+stage=0
+nj=4
+cmd=run.pl
+max_active=7000
+threaded=false
```

Comment:
please remove this option and the modify_ivector_config option, as they are not relevant any more.